docs/book/src/topics/windows.md

```diff
@@ -101,8 +101,29 @@ When creating a cluster with `Machinepool` if the Machine Pool name is longer th
 
 ### VM password and access
 The VM password is [random generated](https://cloudbase-init.readthedocs.io/en/latest/plugins.html#setting-password-main)
-by Cloudbase-init during provisioning of the VM. For Access to the VM you can use ssh which will be configured with SSH
-public key you provided during deployment.
+by Cloudbase-init during provisioning of the VM. For Access to the VM you can use ssh, which can be configured with a
+public key you provide during deployment. For example like this:
+```yaml
+apiVersion: bootstrap.cluster.x-k8s.io/v1alpha3
+kind: KubeadmConfigTemplate
+metadata:
+  name: test1-md-0
+  namespace: default
+spec:
+  template:
+    spec:
+      ...
+      users:
+      - name: username
+        groups: Administrators
+        sshAuthorizedKeys:
+        - "ssh-rsa AAAA..."
+```
+
+Refer to [SSH Access for nodes](ssh-access.md) for instructions on how to add an SSH public key for Windows instances.
```

Comment:
Should this be "for Linux Instances"?

Reply:
I think that sentence is wrong/outdaterd since `ssh-access.md` seems to talk about how to connect to nodes and not about how to add SSH public keys. Will adjust.